buildSrc/src/main/java/saros/gradle/eclipse/CategoryPublisher.java

```diff
@@ -0,0 +1,29 @@
+package saros.gradle.eclipse;
+
+import com.diffplug.gradle.FileMisc;
+import com.diffplug.gradle.eclipserunner.EclipseApp;
+import com.diffplug.gradle.pde.EclipseRelease;
+import com.diffplug.gradle.pde.PdeInstallation;
+import java.io.File;
+
+public class CategoryPublisher extends EclipseApp {
```

Comment:
Add a short javadoc to the class and its method? They don't seem self-explanatory to me.

Reply:
Sure, I will add javadoc